package-lock.json

```diff
@@ -14968,7 +14968,7 @@
     },
     "node_modules/rn-nodeify": {
       "version": "10.3.0",
-      "resolved": "git+ssh://git@github.com/tradle/rn-nodeify.git#338d8d6ba8438403093e9409e9a9d88ad884926f",
+      "resolved": "git+https://github.com/tradle/rn-nodeify.git#338d8d6ba8438403093e9409e9a9d88ad884926f",
```

Comment:
We can't manually edit the auto-generated `package-lock.json`, since it will be overwritten the next time we add something to `package.json`. 
Is there no way to get CI working with git over ssh?

Reply:
True, I didn't want to bump lots of other package versions by regenerating it. I did also update package.json to be explicit about using git+https, so the next time it runs it should resolve the dependency correctly.

Tbh I'm not entirely sure why it was git+ssh in the first place? The [npm i docs](https://docs.npmjs.com/cli/v8/commands/npm-install) aren't very verbose about it, but I would've guessed that you almost always want git+https for any git dep and that it would be the default.

We can get CI working with git+ssh, it would require creating up a public/private key for ssh access and adding to the the github repo config. I wouldn't recommend that though, since all the code is accessible via git+https and having keys will require any fork to also configure keys to get the GitHub actions working. 

The alternatively is to update packages-lock.json?

Reply:
@dmitrizagidulin is anything else needed to get this PR ready?